data/mods/gen2/random-teams.ts

```diff
@@ -115,9 +115,10 @@ export class RandomGen2Teams extends RandomGen3Teams {
 
 	getItem(
 		ability: string,
-		types: Set<string>,
+		types: string[],
```

Comment:
Mainly because I wanted `getItem()` to be consistent with gen 9, no real reason otherwise. While it doesn't matter for item generation, type order can matter for other things like determining the type of Revelation Dance (which in turn affects move enforcement, etc)

Reply:
I suppose that's fine.